### PR TITLE
Feature/ge 684

### PR DESCRIPTION
**Pull request by AltusBarry**

### CODE REVIEW

File: apps/database/docker-entrypoint-initdb.d/init.sql [hunk 1]
```diff
@@ -0,0 +1,5 @@
+CREATE DATABASE accesscontrol;
```
Comment (cobusc) [inline apps/database/docker-entrypoint-initdb.d/init.sql]: The actual databases are called `access_control`, `authentication_service` and `user_data_store` IIRC

File: docker-compose.yml [hunk 2]
```diff
@@ -1,37 +1,95 @@
...
+            context: ../core-authentication-service/
+        volumes:
+            - ../core-authentication-service:/code
+        command: scripts/waitFor.sh scripts/startDjango.sh
+        ports:
+            - "8000:8000"
+        environment:
+            - DB_HOST=172.18.0.2
+            # NOTE: we bootstrap our database container with some dbs.
+            - DB_NAME=authservice
```
Comment (cobusc) [inline docker-compose.yml]: authentication_service

File: docker-compose.yml [hunk 3]
```diff
@@ -1,37 +1,95 @@
...
+            - ALLOWED_HOSTS=*
+            - WAGTAIL_1_IP=http://172.18.0.4:8000/oidc/callback/
+        depends_on:
+            - db
+        networks:
+            oidcnetwork:
+                ipv4_address: 172.18.0.3
```
Comment (cobusc) [inline docker-compose.yml]: One can use the docker compose name to reference containers as well, without explicitly providing IP addresses.
When using it in environment configurations, one can use e.g. `http://core-authentication-service:8000/foo/bar`

**Comment by cobusc** [comment]
@AltusBarry since this is a WIP, just take not of my comments and continue. If you want to merge, let me know and I can +1.

File: docker-compose.yml (hunk 3 above)
Comment (AltusBarry) [reply docker-compose.yml]: This fails for oidc setup, the clients need a static ip which is used for validation and to redirect to. making use of the service names as domains does not work.

File: apps/wagtail_client/wagtail_client/templates/wagtail_client/home.html [hunk 4]
```diff
@@ -1,9 +1,67 @@
...
+{% endblock %}
+{% block extra_js %}
+{{ block.super }}
+<script>
+    // Hard coded like this is fine for demo purposes.
+    // TODO The redirect urls will need to be updated from context later on.
+    document.addEventListener('DOMContentLoaded', function(event) {
+        document.getElementById('redirectButton').addEventListener('click', function(e) {
+            e.preventDefault()
+            var href = 'http://172.18.0.3:8000/registration/';
```
Comment (cobusc) [inline apps/wagtail_client/wagtail_client/templates/wagtail_client/home.html]: Going the extra mile, eh? :)

**Comment by cobusc** [comment]
:+1:

**Pull request closed without merging**